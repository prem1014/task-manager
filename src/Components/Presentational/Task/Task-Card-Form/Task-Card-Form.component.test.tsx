import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import TaskCardForm from './Task-Card-Form.component';

describe('TaskCardForm component test suite', () => {
    Enzyme.configure({ adapter: new Adapter() });
    let componentWrapper: any;
    const taskCardFormProps = {
        taskCardCreated: jest.fn(),
        taskCardUpdated: jest.fn()
    }

    beforeEach( () => {
        componentWrapper = shallow(<TaskCardForm {...taskCardFormProps}/>);
    });

    it('TaskCardForm component snapshot test', () => {
        expect(componentWrapper).toMatchSnapshot();
    });
});