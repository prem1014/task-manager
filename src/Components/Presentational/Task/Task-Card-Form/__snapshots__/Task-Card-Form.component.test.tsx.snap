// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskCardForm component test suite TaskCardForm component snapshot test 1`] = `ShallowWrapper {}`;
